src/mango/src/mango_idx_view.erl

```diff
@@ -199,8 +199,8 @@ opts() ->
             {tag, fields},
             {validator, fun mango_opts:validate_sort/1}
         ]},
-        {<<"selector">>, [
-            {tag, selector},
```

Comment:
I see the get_legacy_selector code above, but reading through code I can't convince myself that removing this option definition won't break old indexes that used the selector name. I wonder if we shouldn't continue to support this for awhile and upgrade it internally for new indexes?

Reply:
And/or perhaps just adding a test specifically for the upgrade would be enough here. And for that I'd like to see us write an old style ddoc directly and then read the index to make sure it behaves as expected.